### PR TITLE
test: Selector playground CT tests

### LINKED ISSUE #19221: Unification App, Test Runner: Write E2E tests around "Runner Header" (including Selector Playground)
### What would you like?

Write end-to-end tests to cover the new Unification work in 10.0-release branch for "[Runner Header](https://docs.google.com/spreadsheets/d/1iPwi89aW6aYeA0VT1XOhYdAWLuScW0okrlfcL9fzh3s/edit#gid=0)" in the App's Test Runner (including Selector Playground). 

### Why is this needed?

_No response_

### Other

_No response_

### PR DESCRIPTION
**Pull request by ryanthemanuel**
- Closes #19221 

### User facing changelog

n/a

### Additional details

This re-enables the selector playground tests and adds tests for :

* The default command and selector in the playground
* The selector playground selector copy functionality

Added percy snapshots for:

* The Spec Runner Header with the selector playground button enabled and disabled
* The Selector Playground 
* The Selector Playground number of elements

### How has the user experience changed?

n/a

### PR Tasks

- [X] Have tests been added/updated?
- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [na] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: packages/app/src/runner/selector-playground/SelectorPlayground.spec.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import SelectorPlayground from './SelectorPlayground.vue'
 
 // TODO: Test is failing due to Percy conflict
 // eslint-disable-next-line
-describe.skip('SelectorPlayground', () => {
+describe('SelectorPlayground', () => {
```
Comment (AtofStryker) [inline packages/app/src/runner/selector-playground/SelectorPlayground.spec.tsx:7]: Are we able to remove the `// TODO: Test is failing due to Percy conflict`?

Comment (AtofStryker) [reply packages/app/src/runner/selector-playground/SelectorPlayground.spec.tsx:7]: I also think the `// eslint-disable-next-line` above is no longer needed. Linter doesn't seem to be complaining 😄.

File: packages/app/src/runner/selector-playground/SelectorPlayground.spec.tsx [hunk 2]
```diff
@@ -20,15 +20,14 @@ describe.skip('SelectorPlayground', () => {
...
-      </div>
-    ))
+  it('populates cy.get by default with a selector of body', () => {
+    const { autIframe } = mountSelectorPlayground()
+
+    cy.spy(autIframe, 'toggleSelectorHighlight')
+    cy.get('[data-cy="playground-method"]').as('method')
+    cy.get('[data-cy="playground-selector"]').as('selector')
+    cy.get('@method').should('contain', 'cy.get')
+    cy.get('@selector').should('have.value', 'body')
```
Comment (tbiethman) [inline packages/app/src/runner/selector-playground/SelectorPlayground.spec.tsx:30]: Nit: could remove the aliases and condense this down, the test ids are pretty descriptive as-is

**Review by tbiethman — APPROVED** [review]

File: packages/app/src/runner/selector-playground/SelectorPlayground.spec.tsx (hunk 2 above)
Comment (ryanthemanuel) [reply packages/app/src/runner/selector-playground/SelectorPlayground.spec.tsx]: Updated here: [`b9d1636` (#19413)](https://github.com/cypress-io/cypress/pull/19413/commits/b9d163621ee8442545e703d273b98f2a8c33c5b3)

File: packages/app/src/runner/selector-playground/SelectorPlayground.spec.tsx (hunk 1 above)
Comment (ryanthemanuel) [reply packages/app/src/runner/selector-playground/SelectorPlayground.spec.tsx:5]: Updated here: [`b9d1636` (#19413)](https://github.com/cypress-io/cypress/pull/19413/commits/b9d163621ee8442545e703d273b98f2a8c33c5b3)

**Review by tbiethman — APPROVED** [review]

**Review by AtofStryker — APPROVED** [review]
🎉

**Pull request merged**